spec/unit/recipes/default_spec.rb

```diff
@@ -5,18 +5,19 @@
 # Copyright (c) 2015 Irving Popovetsky, All Rights Reserved.
 
 require 'spec_helper'
+require 'mixlib/versioning' # new chef-ingredient makes heavy use of this library
```

Comment:
Good point @jtimberman -  I added that line after discovering that my ChefDK didn't include mixlib-versionining and the version I had installed was too old (1.0.0) and kept throwing this error:

```
    NoMethodError:
       chef_ingredient[supermarket] (supermarket-omnibus-cookbook::default line 56) had an error: NoMethodError: undefined method `major' for "0.0.0":String
     # /Users/irving/.chefdk/gem/ruby/2.1.0/gems/mixlib-versioning-1.0.0/lib/mixlib/versioning/format.rb:203:in `<=>'
     # /var/folders/bw/nf13rmz110bf83rqf6658mxm0000gq/T/d20150914-17034-ve4uaq/cookbooks/chef-ingredient/libraries/helpers.rb:49:in `>'
     # /var/folders/bw/nf13rmz110bf83rqf6658mxm0000gq/T/d20150914-17034-ve4uaq/cookbooks/chef-ingredient/libraries/helpers.rb:49:in `block in package_resource'
     # /var/folders/bw/nf13rmz110bf83rqf6658mxm0000gq/T/d20150914-17034-ve4uaq/cookbooks/chef-ingredient/libraries/helpers.rb:46:in `package_resource'
     # /var/folders/bw/nf13rmz110bf83rqf6658mxm0000gq/T/d20150914-17034-ve4uaq/cookbooks/chef-ingredient/libraries/chef_ingredient_provider.rb:40:in `block in <class:ChefIngredient>'
     # ./spec/unit/recipes/default_spec.rb:114:in `block (3 levels) in <top (required)>'
     # ./spec/unit/recipes/default_spec.rb:122:in `block (3 levels) in <top (required)>'
```

The comment should probably have been clearer about the version required.